backend/server/routers/ctf.py

```diff
@@ -133,46 +133,33 @@ def term_sums_even(data: PlannerData) -> bool:
     Check that the sum of the course codes in even terms is even
     """
     is_even: Callable[[int], bool] = lambda x: x % 2 == 0
-    print("Checking even")
-    for y, year in enumerate(data.plan):
-        # Exclude summer term + odd terms
-        for i, term in enumerate(year[2::2], 2):
-            term_sum = sum(map(get_code, term.keys()))
-            print(f"{y}T{i} sum: {term_sum}")
-            if not is_even(term_sum):
-                print("failed: ", term)
-                return False
-
-    return True
+    return all(
+        is_even(sum(map(get_code, term.keys())))
+        for year in data.plan[::2]
+        for term in year[1::2]
+    )
```

Comment:
The fact that you dared to commit the first set of code makes me sad 😭

#forLoopsAreBanned

Reply:
i had a deadline 😭